docs/cow-protocol/tutorials/solvers/routes_integration.md

```diff
@@ -0,0 +1,122 @@
+---
+sidebar_position: 1
+---
+
+# Getting New Routes Integrated Into CoW Protocol
+
+## Overview
+
+This guide is designed for teams building in DeFi who want to get their routes integrated and supported by CoW Protocol. Integration of new liquidity sources enables CoW Protocol's solvers to access more trading opportunities, potentially leading to better prices for users.
+
+## Why integrate with CoW Protocol?
+
+- CoW Protocol uses a unique batch auction mechanism and competition between multiple solvers.
+- Access to a growing user base of traders and integrators
+- Increase your volumes by integrating into the leading trading protocol on Ethereum
+- Finding the best prices across different liquidity sources
+- MEV protection through batch auctions
+- Gas optimization through order batching
+
+## Integration Requirements
+
+### 1. Price Discovery Interface
+
+Solvers need a reliable way to obtain price estimates and available liquidity. This can be provided through:
+
+- On-chain methods to query pool states or oracle prices
+- Off-chain API endpoints providing real-time pricing data
+- WebSocket feeds for continuous updates
+- Documentation of pricing functions or AMM curves
+
+### 2. Settlement Interface
+
+A well-documented smart contract interface for executing trades, including:
+
+- Function specifications for swap execution
+- Required parameters and their formats
+- Gas consumption estimates
+- Safety checks and failure modes
+
+### 3. Technical Documentation
+
+Comprehensive documentation should include:
+
+- Smart contract addresses on supported networks
+- API specifications and endpoints
+- Authentication requirements (if any)
+- Rate limits and other technical constraints
+- Example implementations or code snippets
+
+## Self Service Integration
+
+Instead of waiting for solver teams to have the capacity and resources to deliver your integration, few solver teams offer a self service integration.
+
+**Tycho (Propeller heads) - read more [here](https://docs.propellerheads.xyz/tycho/for-dexs/protocol-integration-sdk)**
+
+## Integration Process
+
+1. **Submit Integration Proposal**
+    - Use the integration proposal template and make sure to add all necessary information.
+    - Post on CoW DAO forum, and engage with any follow-up questions.
+2. **Solver Integration**
+    - Solver teams evaluate integration opportunity
```

Comment:
Inconsistency of "." (dots) use. Some sentences have a dot at the end and some do not